### PR TITLE
Wizard recipe: Jinja2Cpp-1.3.2

### PR DESCRIPTION
**Pull request by AlexKlo**

### CODE REVIEW

File: J/Jinja2Cpp/bundled/patches/update_thirdparty_paths.patch
Comment (giordano) [inline J/Jinja2Cpp/bundled/patches/update_thirdparty_paths.patch:1]: Should this patch be upstreamed?

**Comment by giordano** [comment]
License file is missing

File: J/Jinja2Cpp/bundled/patches/update_thirdparty_paths.patch
Comment (AlexKlo) [reply J/Jinja2Cpp/bundled/patches/update_thirdparty_paths.patch:1]: This patch fixes that issue: https://github.com/jinja2cpp/Jinja2Cpp/issues/260
Maybe we can just wait until it's fixed.

**Comment by AlexKlo** [comment]
> License file is missing

Help, please, what needs to be done to appear?

**Comment by giordano** [comment]
https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file

**Comment by AlexKlo** [comment]
@giordano The license file has been added, so it's possible to start registering the recipe with the patch. Once the issue is resolved, a PR will be submitted to update the script.

**Pull request merged**